solutions/usaco-919.mdx

```diff
@@ -103,4 +103,44 @@ public class paintbarn {
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+data = open('paintbarn.in').read().strip().split('\n')
+n, k = map(int, data[0].split(' '))
+
+coords = [list(map(int, data[i + 1].split(' '))) for i in range(n)]
+
+dp = [[0]*1001 for i in range(1001)]
```

Comment:
i see 1001 everywhere- would it be possible for you to put 1000 in a variable like `WIDTH` and use `WIDTH + 1` everywhere?

Reply:
also uh spaces between operators here